pkg/streamingpromql/testdata/ours/native_histograms.test

```diff
@@ -13,11 +13,93 @@ eval instant at 5m single_histogram
 eval range from 0 to 5m step 1m single_histogram
 	{__name__="single_histogram"} {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:20 count:7 buckets:[9 10 1]}}
 
+# histogram_count extracts the count property from the histogram.
+eval range from 0 to 5m step 1m  histogram_count(single_histogram)
+	{} 4 4 4 4 4 7
+
+# histogram_sum extracts the sum property from the histogram.
+eval range from 0 to 5m step 1m  histogram_sum(single_histogram)
+	{} 5 5 5 5 5 20
+
 clear
 
 # Test metric with mixed floats and histograms
 load 1m
 	mixed_metric 1 2 3 {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:8 count:6 buckets:[1 4 1]}}
 
-eval range from 0 to 4m step 1m mixed_metric
-	{__name__="mixed_metric"} 1 2 3 {{count:4 sum:5 buckets:[1 2 1]}} {{count:6 sum:8 buckets:[1 4 1]}}
+eval range from 0 to 5m step 1m mixed_metric
+	{__name__="mixed_metric"} 1 2 3 {{count:4 sum:5 buckets:[1 2 1]}} {{count:6 sum:8 buckets:[1 4 1]}} {{count:6 sum:8 buckets:[1 4 1]}}
+
+eval range from 0 to 4m step 1m histogram_count(mixed_metric)
+{} _ _ _ 4 6
+
+eval range from 0 to 4m step 1m histogram_sum(mixed_metric)
+{} _ _ _ 5 8
```

Comment:
Is there a reason to stop at 4m in these test cases? Should we make it 5m to match the data above?